po/aa.po

```diff
@@ -0,0 +1,161 @@
+# Afar translation for ubuntu-keyboard
+# Copyright (c) 2015 Rosetta Contributors and Canonical Ltd 2015
```

Comment:
Can we either,

a) s/ubports-keyboard/maliit-keyboard/g

or

b) specify actual license here directly

That way actual license is clear.

Reply:
> a) s/ubports-keyboard/maliit-keyboard/g

Done.